include/eve/arch/cpu/wide.hpp

```diff
@@ -997,6 +997,11 @@ namespace eve
   //! @}
   //================================================================================================
 
+  // Represents a wide that occupies the same size, but formed with int8_t.
+  // Here we use that sizeof(std::int8_t) == 1, so sizeof(In) will represent the number
+  // of std::int8_t, that are needed to have the same size.
+  template<arithmetic_simd_value In> using wide_bytes = wide<std::int8_t, fixed<sizeof(In)>>;
```

Comment:
I am not  convinced we want this typedef, especially in the public api.

Reply:
I can return it back to `rvv_utils`.
I need this typedef for RISC-V as only `int8` (or `uint8`) support full range of lmuls, so only wides with `int8` can be trunc or extend to any needed lmul.

Reply:
I'm so confused by this statement. I thought you cast to u8 and from u8 to write less code. Is that not the reason? Is there some type that is represented with the wrong type.

I' just inline this one. Especially because I'd say that bytes are unsigned.

Reply:
Done